src/compiler/compile/Component.ts

```diff
@@ -324,6 +325,35 @@ export default class Component {
 			js.map.sourcesContent = [
 				this.source
 			];
+
+			if (compile_options.sourceMap) {
```

Comment:
I would probably call this option `sourcemap` (all lowercase) in order to match the rollup option name. I think it'd be easier to remember what to set if they're the same